mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -232,6 +232,7 @@ CLASS net/minecraft/unmapped/C_triydqro net/minecraft/block/AbstractBlock
 	CLASS C_eibemhky AbstractBlockState
 		FIELD f_ciiuhosx opaque Z
 		FIELD f_cmbiwnda hasSidedTransparency Z
+		FIELD f_iazicyxq blockBreakParticles Z
```

Comment:
This is slightly more wordy, but more immediately obvious what it means.

```suggestion
		FIELD f_iazicyxq spawnsParticlesOnBreak Z
```

Reply:
fair enough